<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Алтай-Гид. Клиенты - Отключено</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/logo.png</normaloff>:/images/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableView" name="tableView">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>120</number>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>true</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>794</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_db">
    <property name="title">
     <string>Соединение</string>
    </property>
    <addaction name="action_connect"/>
    <addaction name="action_disconnect"/>
    <addaction name="separator"/>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menu_edit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Редактирование</string>
    </property>
    <addaction name="action_refresh"/>
    <addaction name="separator"/>
    <addaction name="action_add"/>
    <addaction name="action_del"/>
   </widget>
   <widget class="QMenu" name="menu_find">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Поиск</string>
    </property>
    <addaction name="action_find"/>
    <addaction name="action_birthday"/>
   </widget>
   <addaction name="menu_db"/>
   <addaction name="menu_edit"/>
   <addaction name="menu_find"/>
  </widget>
  <action name="action_connect">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Подключиться</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="action_disconnect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Отключиться</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_refresh">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Обновить таблицу</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="action_add">
   <property name="text">
    <string>Добавить строку</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="action_del">
   <property name="text">
    <string>Удалить строку</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
  </action>
  <action name="action_find">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Найти запись</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="action_birthday">
   <property name="text">
    <string>Проверить дни рождения</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
